### PR TITLE
Hardware

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Updated hardware programme page

## QA

- `./run`
- Navigate to /programmes/hardware

## Screenshots

screenshot

### CODE REVIEW

**Comment by anthonydillon** [comment]
@deadlight this PR is targeting master. It should be vanilla-1.8

**Comment by kwm14** [comment]
LGTM @deadlight 👍

File: templates/programmes/hardware.html [hunk 1]
```diff
@@ -4,60 +4,63 @@
...
-            <li>Custom development services</li>
-            <li class="last-item">Product training and competence transfer</li>
-        </ul>
+
+<section class="p-strip is-bordered">
+  <div class="row u-vertically-center">
+    <div class="col-8">
+      <h2 class="p-heading--three">Increase your sales by becoming Ubuntu Certified</h2>
+      <p>Many of the world&rsquo;s biggest PC manufacturers and server providers have chosen to certify their hardware for Ubuntu. Through our Ubuntu Certified programme, we can ensure that your hardware runs perfectly with Ubuntu along with the OEM market leaders who offer Ubuntu Certified hardware.</p>
+      <p><a href="https://certification.ubuntu.com/certification/" class="external">Learn more about Ubuntu certified hardware</a></p>
```
Comment (anthonydillon) [inline templates/programmes/hardware.html]: `p-link--external`

**Comment by deadlight** [comment]
@anthonydillon fixed

**Pull request merged**